### PR TITLE
Update RNHTMLtoPDF.m

### PR DESCRIPTION
**Pull request by wootwoot1234**
Fixes https://github.com/christopherdro/react-native-html-to-pdf/issues/4

### CODE REVIEW

**Comment by christopherdro** [comment]
@wootwoot1234 Thanks for the PR, could you remove the old code entirely instead of commenting it out?

File: RNHTMLtoPDF/RNHTMLtoPDF.m [hunk 1]
```diff
@@ -129,8 +130,9 @@ - (void)webViewDidFinishLoad:(UIWebView *)awebView
         _resolveBlock(_filePath);
     } else {
         NSError *error;
-        _rejectBlock(error);
+        //_rejectBlock(error);
```
Comment (christopherdro) [inline RNHTMLtoPDF/RNHTMLtoPDF.m]: Remove line.

**Comment by wootwoot1234** [comment]
Sorry about that.  It's removed now.

**Comment by christopherdro** [comment]
@wootwoot1234 Thank you!

**Pull request merged**